test/functional/apps/dashboard/new_visualize_flow/index.ts

```diff
@@ -16,12 +16,11 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import { FtrProviderContext } from '../functional/ftr_provider_context';
 
-// eslint-disable-next-line import/no-default-export
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
 export default function ({ loadTestFile }: FtrProviderContext) {
   describe('New Visualize Flow', function () {
-    this.tags('ciGroup2');
```

Comment:
we don't need it since index file is listed in `test/functional/apps/dashboard/index.js`